tests/unit/v1/test_collection.py

```diff
@@ -253,9 +261,9 @@ def test_get(self, query_class):
             get_response = collection.get()
 
         query_class.assert_called_once_with(collection)
-        query_instance = query_class.return_value
-        self.assertIs(get_response, query_instance.stream.return_value)
-        query_instance.stream.assert_called_once_with(transaction=None)
+        query_inst = query_class.return_value
+        self.assertIs(get_response, query_inst.stream.return_value)
+        query_inst.stream.assert_called_once_with(transaction=None)
```

Comment:
I may have overlooked something, but it doesn't look like `query_instance` collides. Is this change for brevity?

Reply:
Looking now, this should be undone. Not sure why this is here honestly. good catch!

Reply:
Thanks!